tests/simulation/params/test_generic_volume_with_sufficient_properties.py

```diff
@@ -0,0 +1,43 @@
+import pytest
```

Comment:
I prefer that these be separated into test_rotation and test_porous_medium, more unit test should be added to each in the future

Reply:
Done